start_ocean.sh

```diff
@@ -264,6 +269,22 @@ while :; do
             export DEPLOY_SUBGRAPH=false
             printf $COLOR_Y'Ocean subgraph will not be deployed, the last deployment (if any) will be intact ...\n\n'$COLOR_RESET
             ;;
+        --predictoor)
+            # Add what we need
+            COMPOSE_FILES+=" -f ${COMPOSE_DIR}/thegraph.yml"
+            # We should remove what is not needed for now, but ocean,py requires both aqua & provider
+            #COMPOSE_FILES="${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/provider.yml/}"
+            #COMPOSE_FILES="${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/redis.yml/}"
+            #COMPOSE_FILES="${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/aquarius.yml/}"
+            #COMPOSE_FILES="${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/elasticsearch.yml/}"
+            COMPOSE_FILES="${COMPOSE_FILES/ -f ${COMPOSE_DIR}\/dashboard.yml/}"
```

Comment:
docker image is being discontinued
https://hub.docker.com/r/portainer/portainer

not sure why this is needed but if possible, it would be great to reduce num of running services